ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.tsx

```diff
@@ -188,7 +188,7 @@ function NetworkGraphPage() {
 
     return (
         <>
-            <PageTitle title="Network Graph" />
+            <PageTitle title="Network Graph (2.0 preview)" />
```

Comment:
@alanonthegit @zhenpesky just wanted to confirm if this is what we want as the page title